src/libraries/Microsoft.Extensions.Configuration.Binder/gen/NullableSpec.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
```

Comment:
Nit: would it be possible to group the "spec" source files into a subfolder so that the model is easier to review?